hadoop-ozone/dist/src/main/compose/ozone-om-ha/docker-config

```diff
@@ -14,8 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-CORE-SITE.XML_fs.o3fs.impl=org.apache.hadoop.fs.ozone.OzoneFileSystem
-CORE-SITE.XML_fs.defaultFS=o3fs://bucket1.volume1.omservice
+CORE-SITE.XML_fs.defaultFS=ofs://omservice
```

Comment:
Why do we need this line to use META-INF services based discovery?

Reply:
I think only MR tests (`ozone-mr` and `ozonesecure-mr`) use `defaultFS` when setting up MR environment:

https://github.com/apache/hadoop-ozone/blob/8102ac799816f417e096873b6f351fd531f7bfbf/hadoop-ozone/dist/src/main/smoketest/createmrenv.robot#L46-L47

Changing `defaultFS` to `ofs://` in these environments can help validate the META-INF changes.

Changing/adding `defaultFS` in the other ones is not needed IMO.

Reply:
AFAIK these are just helper lines, not really the test of the file system. META-INF/services are already tested as test.sh is modified to do both o3fs/ofs tests.

I am fine to replace all the `defaultFS` just let's agree first what do we need to test. Do we need different MR test with both kind of `defaultFs`? 

Or is it enough to test o3fs/ofs low level functionality and MR can use just one of them?

(And you see it's a different discussion, that's the reason why I suggested to do it in a separated PR. But can be included if you think it's better.)

Reply:
> META-INF/services are already tested as test.sh is modified to do both o3fs/ofs tests.

Got it.

> Do we need different MR test with both kind of `defaultFs`?

I'm doing that in HDDS-3840.  I'll pass it via `-D` command-line option, as different value is needed for each test in the same cluster.

I guess I can retain the cluster-level config for convenience.